backend/pkg/Makefile

```diff
@@ -2,10 +2,11 @@ LDFLAGS ?= "-s -w"
 BUILDFLAGS ?= -trimpath -ldflags $(LDFLAGS)
 TESTFLAGS ?=
 
+.PHONY: test
+test:
+	go test $(BUILDFLAGS) $(TESTFLAGS) ./...
+
 .PHONY: generate
 generate:
 	go generate ./...
 
-.PHONY: test
-test:
-	go test $(BUILDFLAGS) $(TESTFLAGS) ./...
```

Comment:
just for me, is this a needed change?

Reply:
I prefer test is the default target rather than `generate`, but maybe we should make a target that executes both as default?